apps/studio/components/interfaces/Settings/Addons/PITRSidePanel.tsx

```diff
@@ -102,10 +102,12 @@ const PITRSidePanel = () => {
 
   const hasReadReplicas = (databases ?? []).length > 1
   const hasChanges = selectedOption !== (subscriptionPitr?.variant.identifier ?? 'pitr_0')
-  const selectedPitr = availableOptions.find((option) => option.identifier === selectedOption)
   const isFreePlan = subscription?.plan?.id === 'free'
+  const selectedPitr = availableOptions.find((option) => option.identifier === selectedOption)
   const blockDowngradeDueToReadReplicas =
     hasChanges && hasReadReplicas && selectedCategory === 'off' && selectedOption === 'pitr_0'
+  const blockDowngradeDueToHipaa =
```

Comment:
Customers with HIPAA should only be able to use 28-days PITR - not less, lowering the PITR days should also not be possible

Reply:
Fixed.